deployment/tools/configMigration.py

```diff
@@ -62,5 +62,7 @@
 # convert service-configuration.yaml
 old_service_configuration = yaml.load(open(os.path.join(input_dir, "services-configuration.yaml")))
 service_configuration, updated = forward_compatibility.service_configuration_convert(old_service_configuration)
+# upgrade image tag version to v0.10.1
+service_configuration["cluster"]["docker-registry"]["tag"] = "v0.10.1"
```

Comment:
Why set this? I think this script are used only when convert old config before cluster object model to new version. @ydye Please make sure this is what you want

Reply:
Ah, this is not cluster object model migration. I misunderstood.